wikibase/queryService/ui/editor/hint/Sparql.js

```diff
@@ -23,62 +23,17 @@ wikibase.queryService.ui.editor.hint = wikibase.queryService.ui.editor.hint || {
 			'hint:Query hint:optimizer "None".',
-			'#TEMPLATE={ "template": { "en": "Textual description of template, referencing ?var" }, "variables": { "?var": { "query": "SELECT ?id WHERE { ?id wdt:P31 wd:Q146. }" } } }'
+			'#TEMPLATE={ "template": { "en": "Textual description of template, referencing ?var" }, "variables": { "?var": { "query": "SELECT ?id WHERE { ?id wdt:P31 wd:Q146. }" } } }'		
 	];
 
 	var SPARQL_PREDICATES = [
-			// wikibase:
-			// property predicates
-			'wikibase:rank', 'wikibase:badge', 'wikibase:propertyType', 'wikibase:directClaim',
-			'wikibase:claim', 'wikibase:statementProperty', 'wikibase:statementValue',
-			'wikibase:qualifier', 'wikibase:qualifierValue', 'wikibase:reference', 'wikibase:referenceValue',
-			'wikibase:statementValueNormalized', 'wikibase:qualifierValueNormalized',
-			'wikibase:referenceValueNormalized', 'wikibase:novalue',
-			// entity types
-			'wikibase:Property', // 'wikibase:Item' disabled on WDQS for performance reasons
-			// data types
-			'wikibase:Reference', 'wikibase:Dump', // 'wikibase:Statement' disabled on WDQS for performance reasons
-			// ranks
-			'wikibase:PreferredRank', 'wikibase:NormalRank', 'wikibase:DeprecatedRank', 'wikibase:BestRank',
-			// value types
-			'wikibase:TimeValue', 'wikibase:QuantityValue', 'wikibase:GlobecoordinateValue',
-			// property types
-			'wikibase:WikibaseItem', 'wikibase:CommonsMedia', 'wikibase:GlobeCoordinate',
-			'wikibase:Monolingualtext', 'wikibase:Quantity', 'wikibase:String', 'wikibase:Time',
-			'wikibase:Url', 'wikibase:ExternalId', 'wikibase:WikibaseProperty', 'wikibase:Math',
-			// pageprops
-			'wikibase:statements', 'wikibase:sitelinks',
-			// time
-			'wikibase:timeValue', 'wikibase:timePrecision', 'wikibase:timeTimezone', 'wikibase:timeCalendarModel',
-			// quantity
-			'wikibase:quantityAmount', 'wikibase:quantityUpperBound', 'wikibase:quantityLowerBound',
-			'wikibase:quantityUnit', 'wikibase:quantityNormalized',
-			// coordinate
-			'wikibase:geoLatitude', 'wikibase:geoLongitude', 'wikibase:geoPrecision', 'wikibase:geoGlobe',
-			// other
-			'wikibase:wikiGroup',
-			//constraints
-			'wikibase:hasViolationForConstraint',
-			// schema: things
-			'schema:about', 'schema:name', 'schema:description', 'schema:dateModified',
-			'schema:Article', 'schema:inLanguage', 'schema:isPartOf',
-			// rdfs: things
-			'rdfs:label', 'rdf:type',
-			// skos: things
-			'skos:altLabel',
-			// xsd:
-			'xsd:dateTime', 'xsd:integer', 'xsd:double', 'xsd:decimal',
-			// geo:
-			'geo:wktLiteral',
-			// owl:
-			'owl:sameAs',
-			// prov:
-			'prov:wasDerivedFrom',
-			// Lexemes
-			'ontolex:LexicalEntry', 'ontolex:Form', 'ontolex:LexicalSense',
-			'ontolex:lexicalForm', 'ontolex:sense', 'ontolex:representation',
-			'wikibase:lemma', 'wikibase:lexicalCategory', 'wikibase:grammaticalFeature',
-			'dct:language'
+            'edm:rights', 'edm:country', 'edm:year', 'edm:language', 'edm:object', 'edm:isShownBy',
```

Comment:
these are again europeana specific, can we have a Sparql.js file in the Europeana folder?